app/Http/Controllers/Users/BulkUsersController.php

```diff
@@ -249,4 +251,38 @@ protected function logItemCheckinAndDelete($items, $itemType)
             $logAction->logaction('checkin from');
         }
     }
+    protected function findAssetOwners()
+    {
+        $asset_owners= Asset::all()->where('assigned_to', '!=', null)->pluck('assigned_to')->toArray();
```

Comment:
This call as it is now would erroneously include the ids of users when it shouldn't, for example if an asset is checked out to a location.